File: 1119-remove-vowels-from-a-string/1119-remove-vowels-from-a-string.cpp
class Solution {
public:
    
    
    
    string removeVowels(string s) {
                
          string ans="";
        
            for(int i=0;i<s.size();i++){
                
                    if(s[i]=='a' or s[i]=='e' or s[i]=='i' or s[i]=='o' or s[i]=='u'){
                        
                                continue;
                        
                    }
                    else{
                        
                            ans+=s[i];
                    }
                
                
            }
        
            return ans;
        
    }
};

File: 1119-remove-vowels-from-a-string/1119-remove-vowels-from-a-string.py
class Solution:
    def removeVowels(self, s: str) -> str:
        
        vowels=('a','e','i','o','u')
        
        ans=''
        
        for char in s:
            
            if char not in vowels:
                    ans+=char
                    
        return ans
    
            
        